### ISSUE TITLE
Create a Hack Nights Page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#403]
### Overview

We are going to move to a model where the hack nights have their own pages when we go back to meeting in person. In the meantime, we need a hack nights page, that says what the status of the hack night is, while maintaining our promotion of the amazing had nights we had.

Details of what the page should contain:
- The images that used to exist on the homepage
- list of projects and icons of those projects by location
- Maybe - links to our last in person meetings via meetup - maybe
- Encouraging people to signup for meetup

### Action Items
- [x] Review the other prototypes
- [x] Design low-fidelity wireframe that mirrors that design but is specific to hack night information
- [x] Draft copy for Hack Nights page
- [x] Revise and proof copy for page
- [x] Design high-fidelity mockup
- [x] Insert copy into finalized mockup
- [ ] Implement new Hack Nights page

### Resources/Instructions
Copy: https://docs.google.com/document/d/1wVbTCuALfUs02qVahkwfAoZ0BSQyYcp_TCbvja0hSrY/edit

[Lucid Chart](https://www.lucidchart.com/pages/)
Prototype the Projects page #156
Prototype the Project-specific page #157

**Comment by ashlingam** [comment]
From @leestella748:
I finally had time to work on the wireframe for the Hack Nights page and had a few questions: 1.  Since we're designing a temporary page due to the coronavirus, should I include other online events such as Happy Hour?
2. Should I include the project team meetings schedule on the Hack Nights page, and if yes, is the posted schedule accurate?  https://www.hackforla.org/project-meetings
3. I'll probably need photos of past events— is there someone I should reach out to/a folder for website-approved assets?

**Comment by ashlingam** [comment]
HI @leestella748, here are my thoughts on the above:
1. Yes, I would include these other online events on the page
2. I'd love to see this in wireframe. Probably makes sense to include, since we want as few clicks as possible to get users to information of interest.
3. Good question. I imagine we have some photos on file, but likely limited. Probably a good question for Bonnie.

**Comment by yuikomajima** [comment]
Here's the sketches we worked on today. 
![IMG_9378](https://user-images.githubusercontent.com/56898767/79698335-27c41800-823d-11ea-8f4c-1972e3bd7172.JPG)
![IMG_9377](https://user-images.githubusercontent.com/56898767/79698341-2c88cc00-823d-11ea-9880-7a4770cb0fa1.JPG)

**Comment by leestella748** [comment]
@harishlingam @yuikomajima 
Here is the wireframe transcribed onto Figma:  https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=153%3A139

Any feedback would be helpful!

**Comment by ExperimentsInHonesty** [comment]
Hi @leestella748 I have many thoughts about several of the designs.  I looked them over last night, and I am still mulling.  I will definitely have an opinion by then, but I have some questions for other people too.  This might be one where we show some options to people outside.  Thanks for coming up with so many different layout options. @harishlingam @yuikomajima

**Comment by ashlingam** [comment]
Hi @leestella748 not sure where you are with your wireframe, but feel free to link them here if you have any initial work that's available for review. Otherwise, no worries since I know we have more to discuss following on Bonnie's comment.

**Comment by ashlingam** [comment]
1. Progress - Discussed revisions to wireframe with Stella, Yuiko, Bonnie, and Harish. Decision on final concept made. Stella to produce high fidelity mockup based on results of earlier discussion. Created draft of copy for page and shared with @myastark for review and input.
2. Blocks - None
3. Availability - Throughout week
4. ETA - 5/2

**Comment by leestella748** [comment]
Hello, I would also like to get the original photos for each location card in order to design a more hi-fi mockup.

**Comment by leestella748** [comment]
@harishlingam @ExperimentsInHonesty @yuikomajima 
Here is what I've worked on so far: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=381%3A214

1. I believe it makes sense to have the COVID-19 blurb up top, as it affects all of our events on the page. I also think it works best when spanned across the entire screen.  
2.  Illustrations vs icons?
3.  I'm not sure where to get the original photos for each location card, so if someone could share them with me that would be great!

The first one on the left is most similar to what we discussed last time, but my favorite iteration is in the middle. Let me know what you think!

**Comment by ExperimentsInHonesty** [comment]
@leestella748 fantastic new designs.  @harishlingam and I choose design #1 (on left), with the slight modification of adding the days of the week to the locations section, and ordering them this way
Monday - Westside
Tuesday - DTLA
Wednesday - South LA

**Comment by ExperimentsInHonesty** [comment]
- [x] @ExperimentsInHonesty will provide images for the location cards
- [x] @leestella748 will add the images
- [ ] @harishlingam and @ExperimentsInHonesty  will review the copy

**Comment by ExperimentsInHonesty** [comment]
@leestella748 please make each of the project teams on the team meeting schedule be red underlined, then those will link to the projects.

**Comment by ExperimentsInHonesty** [comment]
Here is where the images are located: https://github.com/hackforla/website/tree/gh-pages/assets/images/hack-nights

**Comment by leestella748** [comment]
@ExperimentsInHonesty @harishlingam Here's the updated wireframe: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=369%3A212

**Comment by ashlingam** [comment]
Hi @leestella748 Bonnie and I had a chance to review the mockup. I took the liberty of making a few minor edits. Here is our feedback:

- Please make the days of week more prominent, since they look too small.
- In the Downtown LA Location section, you will note the grey containers that have been added to three of the projects. Please replicate for across all projects for all locations, and increase spacing between the rows of projects as appropriate.
- We've tweaked the top of the page to now show a screen grab of the referenced Ted Talk. Feel free to adjust the styling here if compelled.

For clarification, I've added comments to the Figma file.

**Comment by leestella748** [comment]
@harishlingam I made the changes: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=445%3A0

**Comment by leestella748** [comment]
@harishlingam @ExperimentsInHonesty 
Let me know if there are any additional changes you'd like to see!  

Figma:  https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=545%3A2806

Prototype view: https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=545%3A2806&viewport=-4611%2C-10732%2C0.6133910417556763&scaling=scale-down-width

**Comment by ExperimentsInHonesty** [comment]
@leestella748 @yuikomajima @harishlingam - I made a few minor changes (added some text, etc).   This is ready to send to development, so I will put this in the prioritized backlog.

![HiFi](https://user-images.githubusercontent.com/37763229/81742266-b13dc300-9454-11ea-999a-890b819ad8f7.png)

**Comment by ExperimentsInHonesty** [comment]
@leestella748 is going to make a mobile version mockup.

**Comment by leestella748** [comment]
Here is the mobile version: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=705%3A999

**Comment by ExperimentsInHonesty** [comment]
@leestella748 is going to rethink the mobile version to make it less scrolly

**Comment by leestella748** [comment]
Here are two versions: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=887%3A1

The first version uses separate tabs, which allows users to switch between "Events" and Locations" without so much scrolling.  However, I'm not sure if clicking on a tab will cause the whole page to reload or if we can have both tabs load on open.  

For the second version, each section can be collapsed or expanded.

**Comment by leestella748** [comment]
Mobile wireframes: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=961%3A181
Mobile prototype: https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=926%3A2438&viewport=-2382%2C-8117%2C0.29358410835266113&scaling=min-zoom

**Comment by ExperimentsInHonesty** [comment]
@leestella748  we get rid of the white line at the bottom before the footer (on the mobile)?

**Comment by leestella748** [comment]
@ExperimentsInHonesty Done.  However, the mobile version of our Getting Started page has the white line, so if we want to stay consistent we should have someone go back in and reduce the margin.

**Comment by ExperimentsInHonesty** [comment]
@leestella748 indeed it does.  I’ll put removing that in a new issue.

**Comment by ExperimentsInHonesty** [comment]
@leestella748 ok, so I was wrong to say to remove that white section, because its literally everywhere (all the project pages), etc.  Sorry! Can you put it back?

**Comment by leestella748** [comment]
@ExperimentsInHonesty Done!  Anything else that you'd like changed before it's ready for the dev team?

**Comment by ExperimentsInHonesty** [comment]
Nope that it.  I’ll find out who is looking for something good.  And I’ll go to the prioritized backlog and make sure it’s actually prioritized so you can start on next!

**Comment by ashlingam** [comment]
@leestella748 @ExperimentsInHonesty This looks great! If I could add one minor comment, I would reduce the weight of font in the white drop down buttons to medium (i.e. "Meetup Events" "Online Project Team Meetings").

**Comment by leestella748** [comment]
@harishlingam Done!  

I'll put links here for the dev team:
Desktop & mobile wireframes: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1028%3A2382
Mobile prototype view: https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=926%3A2438&viewport=-459%2C-3206%2C0.1227351650595665&scaling=min-zoom

**Comment by leonelRos** [comment]
I did not have a lot of time this past week to work on it. So, I do not have any updates.

**Comment by ExperimentsInHonesty** [comment]
@leonelRos please provide details about how much availability you will have in the coming week.  And ETA.

Generally the format is 
1. Progress
2. Blocks
3. Availability
4. ETA

**Comment by leonelRos** [comment]
Progress: only worked 10% of it
Blocks: time-limited.
Availability: Wednesday to Friday
ETA: Sunday 06-21-2020

**Comment by leonelRos** [comment]
Progress: working with the latest mock
Blocks: time-limited.
Availability: Wednesday to Friday
ETA: Tuesday 06-23-20

**Comment by leonelRos** [comment]
Hey team here is my progress

Progress: working on the last part where I need to populate the projects for each hack for LA location. The page is 90% ready and will be posting images in the issue number.
Blocks: limited time.
Availability: Wednesday to Friday
ETA: Tuesday 06-29-20

**Comment by leonelRos** [comment]
Progress: working on styling and adding mobile versions.
Blocks: limited time.
Availability: Wednesday to Friday
ETA: Sunday 07-05-20 end of the day.

**Comment by leonelRos** [comment]
<img width="1370" alt="Screen Shot 2020-07-12 at 9 21 58 AM" src="https://user-images.githubusercontent.com/50648001/87251515-24e25900-c421-11ea-9352-5d6a0f738bee.png">
<img width="1121" alt="Screen Shot 2020-07-12 at 9 24 28 AM" src="https://user-images.githubusercontent.com/50648001/87251571-7ee31e80-c421-11ea-89ad-6f914d11286b.png">
<img width="1165" alt="Screen Shot 2020-07-12 at 9 24 48 AM" src="https://user-images.githubusercontent.com/50648001/87251577-8a364a00-c421-11ea-9b47-f85d20769f7a.png">

--Mobile version

<img width="201" alt="Screen Shot 2020-07-12 at 9 25 10 AM" src="https://user-images.githubusercontent.com/50648001/87251587-97ebcf80-c421-11ea-8495-4bb20a7c9c2c.png">
<img width="199" alt="Screen Shot 2020-07-12 at 9 25 40 AM" src="https://user-images.githubusercontent.com/50648001/87251606-a934dc00-c421-11ea-9b0f-e2f3f58748e5.png">
<img width="202" alt="Screen Shot 2020-07-12 at 9 26 27 AM" src="https://user-images.githubusercontent.com/50648001/87251618-c669aa80-c421-11ea-9b8a-3c129187a2ae.png">
<img width="170" alt="Screen Shot 2020-07-12 at 9 27 35 AM" src="https://user-images.githubusercontent.com/50648001/87251637-eef1a480-c421-11ea-9b48-9859f977a81e.png">

Will show live version!!!!

**Comment by alexandrastubbs** [comment]
@leonelRos a helpful tool to capture your full screen is this [google chrome extension](https://chrome.google.com/webstore/detail/full-page-screen-capture/fdpohaocaechififmbbbbbknoalclacl?hl=en)

**Comment by alexandrastubbs** [comment]
@leestella748 will update Figma to show what happens when the 'our locations' is dropped down. @leonelRos will update spelling for 'Hack Nights'.

**Comment by leestella748** [comment]
@leonelRos done 
<img width="305" alt="image" src="https://user-images.githubusercontent.com/61727650/87252980-1f3e4080-c42c-11ea-9fe2-5dcc497b6490.png">

**Comment by leonelRos** [comment]
Thanks, @leestella748.
I refactored most
 of the styling and now the page looks like that.

![screencapture-localhost-4000-hack-nights-page-2020-07-12-11_48_52](https://user-images.githubusercontent.com/50648001/87254214-bbb91080-c435-11ea-9f63-d871423e7344.png)

**Comment by leestella748** [comment]
@ExperimentsInHonesty  During the breakout room with Alex and Yuiko, we also discussed the hack nights mobile page.  The original plan was to keep the “see our locations” dropdown within the yellow COVID-19 banner, but we agreed that it made more sense to keep the items separate.  I talked to Leo as well since he’s working on the ticket, and he agreed and said he actually already has a second version that reflects this.   

Clickable prototype: https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1712%3A3408&viewport=-868%2C-3622%2C0.13307054340839386&scaling=min-zoom

Figma: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1712%3A3613

<img width="250" alt="image" src="https://user-images.githubusercontent.com/61727650/87256144-1efe6f00-c445-11ea-8c0e-e38cc8561e57.png">

**Comment by alexandrastubbs** [comment]
@leestella748 some feedback to share after chatting w/ Bonnie. Happy to voice over live on Sunday. 

- Given the importance of the COVID message and wanting to make sure that individuals understand that the locations are not accessible right now, we would like the locations pill to live within the yellow and upon expansion, want the yellow background to expand with the content that it sits behind
- Would like the different days to be separate bubbles and not merged together in a single white bubble 
- OK with 'See our locations' being a separate bubble, and then all of the white bubbles can be in front of the yellow (suggesting their connection to each other and the location CTA)

**Comment by alexandrastubbs** [comment]
Discuss name for page in today's meeting.

**Comment by alexandrastubbs** [comment]
Name to be changed to Events.

**Comment by leonelRos** [comment]
Hey guys I didn't have much time to finish the new mock up this week but I will be working on it all day today. 


Progress: web and mobile version, need to add functionality 
Blocks: time-limited.
Availability: Wednesday to Friday
ETA: 07-28-20

**Comment by leonelRos** [comment]
Hey @alexandrastubbs  @ExperimentsInHonesty, I finished the page and it is ready for PR. I took screenshots of every version. Feel free to comment or give feedback if anything needs to change. 

![screencapture-localhost-4000-hack-nights-page-2020-08-02-08_54_10](https://user-images.githubusercontent.com/50648001/89126815-b59ada80-d49d-11ea-8de0-1f363baf0a7f.png)


![screencapture-localhost-4000-hack-nights-page-2020-08-02-08_55_02](https://user-images.githubusercontent.com/50648001/89126827-cf3c2200-d49d-11ea-9aed-94aab8bc7908.png)


![screencapture-localhost-4000-hack-nights-page-2020-08-02-08_55_45](https://user-images.githubusercontent.com/50648001/89126843-f397fe80-d49d-11ea-9680-c43737f92471.png)

**Comment by alexandrastubbs** [comment]
@leonelRos when are you available to review this live? We have some questions we would like to address.

**Comment by leonelRos** [comment]
@alexandrastubbs I should be able to meet up today around 7 or 8 PM PT. Let me know if that is ok with both, if not, we can meet on Tuesday at developers meeting.

**Comment by alexandrastubbs** [comment]
@leonelRos we are working with Stella to identify the discrepancies in the font sizes, because it appears that there are quite a few different sizes in both desktop and mobile version. In the meantime, we have two other points of feedback that we'd like to update before pushing this live: 

1. The line spacing is not correct. Can you update to look more like the Figma? For example, where you have Meetup Events, there are huge spaces between the sub heads where they don't exist in the figma. 
2. By video at the top of the mobile version, the arrow pointing the wrong direction. Reference the Figma for correct orientation.

**Comment by leonelRos** [comment]
@alexandrastubbs, Thank you for pointing that out. Since you guys are checking on some other details. I would like to point that out that the main reason why I have those spaces in "Meetup events" is to cover the height of the "Online project teams meetings" bubble. The calendar part populates itself by calling the script from Project teams meeting page https://www.hackforla.org/project-meetings. I will make it look more fitting to the Figma file and that would mean adding more properties to the Scss file.  This is what it would look like with the following changes. 

![screencapture-localhost-4000-hack-nights-page-2020-08-07-15_50_57](https://user-images.githubusercontent.com/50648001/89694561-e330bb00-d8c5-11ea-865d-bb3f95dc7101.png)

**Comment by alexandrastubbs** [comment]
@leonelRos can you share mockups of the mobile version as well?

**Comment by leonelRos** [comment]
@alexandrastubbs, it does not change much since mobile version has its own properties.

![screencapture-localhost-4000-hack-nights-page-2020-08-09-09_59_56](https://user-images.githubusercontent.com/50648001/89737614-1179dc00-da27-11ea-9cd5-b811d4686c82.png)

**Comment by leonelRos** [comment]
Hey guys I didn't have much time this week. I have the header to look exactly like figma file

Progress: web and mobile version need adjustments.
Blocks: time-limited.
Availability: Wednesday to Friday
ETA: 08-18-20

**Comment by leonelRos** [comment]
Progress: web and mobile versions need adjustments.
Blocks: time-limited, redo the header.
Availability: Friday and saturday
ETA: 08-23-20

**Comment by leonelRos** [comment]
Hey Team, I got the design as displayed in the figma file, I still working on the arrow vectors, it won't rotate as a commanded on my code, so still working on that!

![screencapture-localhost-4000-hack-nights-page-2020-08-29-20_11_00](https://user-images.githubusercontent.com/50648001/91650403-24833900-ea34-11ea-8971-20e53e9f8d81.png)

![screencapture-localhost-4000-hack-nights-page-2020-08-29-20_09_45](https://user-images.githubusercontent.com/50648001/91650404-2816c000-ea34-11ea-8ee2-7457dad999ac.png)

**Comment by leonelRos** [comment]
Hey team, I still debugging the arrows. Susan and I worked on it last Thursday but we could not figure it out. This is the error I am getting:

1. The page looks fully complete.![image](https://user-images.githubusercontent.com/50648001/92330658-61978e80-f025-11ea-82b8-df536d0cc430.jpeg)

2.After going to mobile version and clicking the arrows for the drop down and go back to full desktop page we see the arrows in those two bubble cards.
![image](https://user-images.githubusercontent.com/50648001/92330689-9c012b80-f025-11ea-8694-75169370c3b2.jpeg)

I will try to make sure this is done by developers meeting!

**Comment by ExperimentsInHonesty** [comment]
@leonelRos Let's go ahead and push the page and fix the problem after.  Also, we could use the live page to ask for help in the general channel which might get some more developers on the project.

**Comment by leonelRos** [comment]
@ExperimentsInHonesty  OK, I will create the PR and assign @ye-susan to review it.

**Comment by alexandrastubbs** [comment]
@leonelRos We just looked at the [pull request](https://github.com/hackforla/website/pull/722) - who/what is the blocker now?

**Comment by alexandrastubbs** [comment]
@leonelRos any updates on this?

**Comment by ExperimentsInHonesty** [comment]
@cnk will fix meeting times not loading and add timezone lang
![Screen Shot 2020-09-20 at 11 24 00 AM](https://user-images.githubusercontent.com/37763229/93718887-e01b2280-fb33-11ea-8286-47e4244456a4.png)


Issues that need to be looked at that are unassigned currently
1. The top white header needs to fill the page like the toolkit page does
https://www.hackforla.org/hack-nights-page
https://www.hackforla.org/toolkit
![Screen Shot 2020-09-20 at 10 52 38 AM](https://user-images.githubusercontent.com/37763229/93718207-739e2480-fb2f-11ea-9cf6-321ec584a83a.png)

2. Why is there any pink at all

3. The text for the video is not responsive and is merging with the about text
![Screen Shot 2020-09-20 at 10 55 06 AM](https://user-images.githubusercontent.com/37763229/93718277-dabbd900-fb2f-11ea-845f-1edb09521b75.png)

**Comment by cnk** [comment]
I renamed this so the url will be hackforla.org/hack-nights and made the meeting times section work. Also added the Time Zone note from the current meeting-times pages.

**Comment by ExperimentsInHonesty** [comment]
@drubgrubby will take care of the remaining issues here: https://github.com/hackforla/website/issues/403#issuecomment-695816107

**Comment by drubgrubby** [comment]
I made the fixes mentioned in https://github.com/hackforla/website/issues/403#issuecomment-695816107 .  
1) The white background fills the header and the "pink" is gone.
2) The text for the video is now responsive and doesn't merge with the about text. 
3) In the process, because there was a header-container tag that was being used from getting-started-page.scss, I created an ../elements/_containers.scss file and put the header-container tag there so it can logically be used on whatever page wants it now.  (This was done in consultation with @cnk )

<img width="1221" alt="Screen Shot 2020-09-24 at 1 20 32 PM" src="https://user-images.githubusercontent.com/68244054/94197093-25a85a00-fe6a-11ea-878e-a362c8ee2882.png">

**Comment by drubgrubby** [comment]
I have not made a pull request on this because the mobile version doesn't look quite right yet.  I'm not sure if I should include that in the same pull request, or merge this and make the mobile version a separate issue/PR. 

Option A: Do the fixes to the mobile and include it in the same PR
Option B: Do a PR for the current work and make a separate issue/PR for the mobile site.

**Comment by cnk** [comment]
Either is fine. If you don't have a lot of time, perhaps pushing this now. Even without mobile fixes it is certainly better than the current version.

**Comment by ExperimentsInHonesty** [comment]
#748 also related
